python/ray/data/_internal/progress_bar.py

```diff
@@ -83,6 +87,12 @@ def __init__(
                 needs_warning = False
             self._bar = None
 
+    def _truncate_name(self, name: str) -> str:
```

Comment:
Should we add a warn-once that the name is getting truncated and that the behavior can be disabled with `DEFAULT_ENABLE_PROGRESS_BAR_NAME_TRUNCATION`? Not sure if users will know how to disable it otherwise